### PR TITLE
first beard vanity inclusion

### PR DESCRIPTION
**Pull request by cloudziq**
someone need to add its config to data/vanities.cfg  (included in commit)

### CODE REVIEW

File: beard/CONFIG LINE.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+addvanity 5     "beard"                 "beard"             "tag_moustache"     3   2       // face
```
Comment (qreeves) [inline beard/CONFIG LINE.txt]: Remove this and just put it in the description for the pull request.

Comment (cloudziq) [reply beard/CONFIG LINE.txt]: ok, but I don't understand why, you can't add this and remove file ?   lazy ?

**Comment by Dziq84** [comment]
just messed things because crappy net, thanks

**Comment by cloudziq** [comment]
Quin, why you cant add this line and delete file later?   I dont get it.. now I messed this PR so much ;p anyway, this is that config.....   :

addvanity 5     "beard"                 "beard"             "tag_moustache"     3   2       // face

**Comment by qreeves** [comment]
Because you shouldn't add files to the production repository if they're not intended to be part of the game. This isn't a scratch space, that's what the `community` repository is for.

**Comment by cloudziq** [comment]
I know I should not add files like that :-) but you can simply remove that "dummy" file, or not ?  ;>

**Pull request merged**